src/Steps/TOInfo.tsx

```diff
@@ -14,14 +14,15 @@ export const TOInfo = () => {
         <Col>
           <Form.Control required type="text"
             value={globalState.wizardOptions[globalState.tmcrIndex].program_mod_system_name}
-            onChange={e => dispatch({ type: 'MERGE_OPTION', payload: {'program_mod_system_name': e.target.value}})} />
+            onChange={e => dispatch({ type: 'MERGE_OPTION', payload: {'program_mod_system_name': e.target.value}})}
+            readOnly={globalState.tmcrIndex === 1} />
```

Comment:
@RobertPorterfield - These "shared" fields can be a source of bugs in the application.  Suggest they either be moved outside of the TMCR index object and brought up to a higher level, or instead of pulling from globalState.tmcrIndex pull from the first one always.  These applies to all of the shared fields on the different components.

Reply:
I agree with the concept and the bugs. Will have to test this with the docxtemplater does its parsing. Not sure if I can reference a higher level variable from within an array.